scripts/install_dev_server.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
```

Comment:
This turned out to be substantially more complicated than I thought it would be.  In order to merge them into one script I had to alter the behavior of a few things since it is now called in 4 ways.

From config/debian/dpkg_server/rules - It should still behave the same.
Dockerfile - I had to alter the Dockerfile to let the script do more of the work. (Can you test to make sure this actually produces a system that matches what you were trying to set up? It shuffles the ordering of how it gets some things done, but I think it brings up an identical system)
Vagrantfile - I had to alter the Vagrantfile to pass args to use symlinks and point at a different src_dir
directly when installing GRR from src - The script now acts as an all in one that installs all dependencies

When this script installs dependencies it uses the requirements.txt from the source directory instead of relying on a separately hosted file that would most likely be out of date for a from source install, especially for a devel branch.